db/seeds/users.yml

```diff
@@ -44,3 +44,17 @@ whats-new-private-beta-completed@example.com:
   last_name: DemoUser
   setting_type: 'school'
   # optional
+
+brett@example.com:
```

Comment:
Is seed Brett in specs?

Reply:
not sure what you mean? I created so I always have a user I can test with that doesn't conflict with complete@

Reply:
Conflicting in review apps with many testers, not local development, right?